plutus-core/plutus-ir/test/prettyprintingReadable/stupidZero.golden

```diff
@@ -0,0 +1,9 @@
+letrec data Nat | match_Nat where
+         Zero : Nat
+         Suc : Nat -> Nat
+in
+let !three : Nat = Suc (Suc (Suc Zero)) in
```

Comment:
I think this `in` would ideally be on the next line, but I think that would require being fairly clever.

Reply:
Yeah this is a tricky one because you want one-liners when you don't have a surrounding context of lets. Maybe not worth the extra complexity in the code.

Reply:
If one wanted to spend unlimited time tweaking this one could also consider doing something like:
```
let x : t = e
in let
     y : t = e
in let
     z : t = e
in foo x y z
```
or
```
let     x : t = e
in let y : t = e
in let z : t = e
in foo x y z
```
or ...

Reply:
But I think we are at a good point with this where the marginal value of another hour spent on this is lower than a bunch of other stuff that needs doing so I'm pretty happy.